mnefun/_mnefun.py

```diff
@@ -685,7 +687,7 @@ def fetch_raw_files(p, subjects, run_indices):
             cmd += ['--include', op.basename(fname)]
         remote_loc = '%s:%s' % (p.acq_ssh, op.join(remote_dir, ''))
         cmd += ['--exclude', '*', remote_loc, op.join(raw_dir, '')]
-        run_subprocess(cmd)
+        run_subprocess(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         # move files to root raw_dir
```

Comment:
[Python pages](https://docs.python.org/2/library/subprocess.html#using-the-subprocess-module) recommends 

> Do not use stdout=PIPE or stderr=PIPE with this function as that can deadlock based on the child process output volume. Use Popen with the communicate() method when you need pipes.

You thinking this use case is simple enough to ignore this?

Reply:
I think it should be okay